app/src/proxy.ts

```diff
@@ -0,0 +1,71 @@
+import child_process from "node:child_process";
+
+export type ProxyRequest = {
+  method: string;
+  path_query: string;
+  stream: false;
+  body: string;
```

Comment:
I thought we would have needed to do `body?` to mark it as optional but it seems fine being omitted in App.tsx...not sure what I'm missing.

Reply:
ooh good catch!! it looks like when we use the `as Type` type assertion, typescript skips compile-time type checks. will remove that + construct the type the standard way, but also mark `body` optional here